test/Makefile.autosetup

```diff
@@ -467,6 +469,7 @@ TEST_OBJS	= test/main.o \
 		  $(MBYTE_OBJS) \
 		  $(MD5_OBJS) \
 		  $(MEMORY_OBJS) \
+		  $(MISC_OBJS) \
```

Comment:
I am merely suggesting this, but I can get rid of it if you don't think this is necessary.